packages/amplify-util-mock/src/api/api.ts

```diff
@@ -381,14 +384,20 @@ export class APITest {
       return undefined;
     });
     if (!name) {
-      throw new AmplifyFault('MockProcessFault', {
+      throw new AmplifyError('MockProcessError', {
         message: 'No AppSync API is added to the project',
-        link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url,
+        resolution: `Use 'amplify add api' in the root of your app directory to create a GraphQL API.`,
+        link: `${errorSuffix}`,
       });
     }
     return name;
   }
 
+// extracted method for testing purposes only
+  public async getAppSyncApiPublic(context) {
+    return await this.getAppSyncAPI(context);
+  }
+
```

Comment:
If you need this function exported for testing, I would just make the original method `public` rather than adding another a new pubic wrapper method.